cPP_DSC.adoc

```diff
@@ -4019,203 +4015,391 @@ The dependencies between SFRs implemented by the TOE are addressed as follows.
 |Rationale Statement
 
 |FCS_CKM.1 
-|[FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_COP.1] 
+|[_FCS_CKM.1/AKG_, _FCS_CKM.1/SKG_, FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7, _FCS_CKM_EXT.8_ or FCS_COP.1] 
```

Comment:
FCS_CKM_EXT.7 is not in CC:2022, is it? I also don't see it being required by us in FCS_CKM.1

Reply:
In the crypto catalog they have it listed, since clearly it can't be there from cc:2022 as you say. Similarly FCS_CKM.1 can't be a dependency on itself since it would be circular in the line below (but that is how we wrote the requirement).

Basically the list of dependencies is a combination of the CC:2022 and the crypto catalog (plus, where applicable, what we may have added)